ai/Util/ball_possession.py

```diff
@@ -9,10 +11,10 @@
 def can_get_ball(game_state, player_id, target):
     player_position = game_state.get_player_pose(player_id).position
     ball_position = game_state.get_ball_position()
-
+    player_orientation = game_state.get_player_pose(player_id).orientation
     if is_inside_circle(player_position, ball_position, RADIUS_TO_GRAB_BALL):
 
-        if is_facing_point_and_target(player_position, ball_position, target, ANGLE_TO_GRAB_BALL):
+        if fabs(player_orientation - get_angle(player_position, ball_position)) <= ANGLE_TO_GRAB_BALL:
```

Comment:
Il y a un bug wrap to pi si player_orientation est pi et get_angle() retourne -pi. Simon devrait avoir réglé cela sur sa branche (checker avec lui).
Aussi, ANGLE_TO_GRAB_BALL est de 1 Rad. Ce que est pas mal gros